content/rel/_index.md

```diff
@@ -10,14 +10,14 @@ Any valid rel value can be used in WebFinger links.
 
 The following rel values have been defined in the webfinger.net namespace to address common WebFinger use cases.
 
-### `http://webfinger.net/rel/avatar` {#avatar}
+### `https://webfinger.net/rel/avatar` {#avatar}
```

Comment:
I understand the urge to change a URL to https, but the problem here is, that the `http` version is widely spread and because this is an ID, I would not change it: https://www.packetizer.com/ws/webfinger/link_relations.html

@willnorris what do you think?

Reply:
Thanks for the pointer, @pfefferle. 

So it seems that using an outdated plain text protocol identifier is set in stone forever, and changing it to the encrypted protocol identifier that is ubiquitous in the year (Snowden+11) is a risk to a base of the fediverse. 

It looks fine as long as we still serve our sites through a protocol that [uses 'Referer:' in its requests](https://en.wikipedia.org/wiki/HTTP_referer), because nobody dared to fix that typo back then.

Reply:
Hello @pfefferle!

Thanks for the deep insight into your thoughts! 

I am not offended at all - quite the opposite ;) Methinks I understand the limitations you probably have to adhere to, and I firmly believe that backwards compatibility is a "Type A" priority for anything involving such a large number of participants. 

It just seems a bit puzzling to me that a document that is central to the future of our web, as opposed to the surveillance capitalist swamp, contains mission-critical details that are subject to change due to technical progress, but that incorporating that inevitable progress into the document would break the subject of said standard. 

OTOH, as TimBL said: ["Cool URIs don't change"](https://www.w3.org/Provider/Style/URI.html) so you're probably on the right track, and it could be an integral part of the standard that the URIs it contains don't change ad infinitum.

Just saying.

Thanks, Robert.

Reply:
@pfefferle I will adjust the PR accordingly. Please hang on.

Reply:
Done. That went well.

Reply:
Vielen Dank :)